h2o-algos/src/main/java/hex/ensemble/Metalearners.java

```diff
@@ -209,6 +209,9 @@ protected void setCustomParams(GLMParameters parms) {
             //add GLM custom params
             super.setCustomParams(parms);
 
+            parms._generate_scoring_history = true;
+            parms._score_iteration_interval = (parms._valid == null) ? 5 : -1;
```

Comment:
`scoring_iteration_interval` might change depending on benchmark results.
 
 When `_valid` is specified, we use lambda search which provides plenty of information for the learning curve. Otherwise, lambda search is off and the information in scoring history is often just one or two points entries. Even metalearner trained on Airlines dataset subset with 250k rows has less than 10 iterations so even this `score_iteration_iterval` might be too big.

Reply:
With benchmark results from @wendycwong we decided to keep the value to 5 as it has less than 2% performance impact but it improves the learning curve in some situations significantly. This affects only the AUTO metalearner in SE.

<img width="567" alt="Screen Shot 2021-03-22 at 20 11 48" src="https://user-images.githubusercontent.com/61695433/112045261-f047a500-8b4a-11eb-9c0f-567883257b32.png">